lib/msal-common/src/client/BaseClient.ts

```diff
@@ -52,7 +52,7 @@ export abstract class BaseClient {
         // Set the network interface
         this.networkClient = this.config.networkInterface;
 
-        B2cAuthority.setKnownAuthorities(this.config.authOptions.knownAuthorities);
+        TrustedAuthority.setTrustedAuthoritiesFromConfig(this.config.authOptions.knownAuthorities, this.config.authOptions.instanceMetadata);
```

Comment:
Do trusted authorities have to be set everytime a base client is created? Can they just be set once when the client application and authority are created?

Reply:
I put it here so we'd only need to do it in one place as all the other clients extend this one, making it easier when we change behavior or add new clients. The first thing this function does is checks if the internal `instanceMetadata` list has already been set and won't do anything if it has. So it should only set trusted authorities once, but will check if it needs to each time base client is created.